README.md

```diff
@@ -183,74 +181,80 @@ bases:
-Kubernetes components:
-
-- on-premise or unmanaged cloud clusters
-- Google Kubernetes Engine (GKE)
-- Azure Kubernetes Service (AKS)
-- Elastic Kubernetes Service (EKS)
-- OVH Kubernetes Service
+``` yaml
+  ...
+  - ./vendor/katalog/monitoring/eks-sm
 
-### On-premise or unmanaged cloud clusters
+```
 
-- Add `monitoring/kubeadm-sm` and `monitoring/configs` to `Furyfile.yml`.
-- Download package with `furyctl vendor`
-- Add `./vendor/katalog/monitoring/kubeadm-sm` to `kustomization.yaml`.
-- Deploy package with `kustomize build . | kubectl apply -f -`
+- For GCP GKE
 
-### Google Kubernetes Engine (GKE)
+``` yaml
+  ...
+  - ./vendor/katalog/monitoring/gke-sm
 
-- Add `monitoring/gke-sm` and `monitoring/configs` to `Furyfile.yml`.
-- Download package with `furyctl vendor`
-- Add `./vendor/katalog/monitoring/gke-sm` to `kustomization.yaml`.
-- Deploy package with `kustomize build . | kubectl apply -f -`
+```
 
-### Elastic Kubernetes Service (EKS)
+- For Azure AKS
 
-- Add `monitoring/eks-sm` and `monitoring/configs` to `Furyfile.yml`.
-- Download package with `furyctl vendor`
-- Add `./vendor/katalog/monitoring/eks-sm` to `kustomization.yaml`.
-- Deploy package with `kustomize build . | kubectl apply -f -`
+``` yaml
+  ...
+  - ./vendor/katalog/monitoring/aks-sm
 
-### Azure Kubernetes Service (AKS)
+```
 
-- Add `monitoring/aks-sm` and `monitoring/configs` to `Furyfile.yml`.
-- Download package with `furyctl vendor`
-- Add `./vendor/katalog/monitoring/aks-sm` to `kustomization.yaml`.
-- Deploy package with `kustomize build . | kubectl apply -f -`
+- For On-premise/self-hosted
 
-### OVH Kubernetes Service
+``` yaml
+  ...
+  - ./vendor/katalog/monitoring/kubeadm-sm
 
-- Add `monitoring/ovh-sm` and `monitoring/configs` to `Furyfile.yml`.
-- Download package with `furyctl vendor`
-- Add `./vendor/katalog/monitoring/ovh-sm` to `kustomization.yaml`.
-- Deploy package with `kustomize build . | kubectl apply -f -`
+```
 
-If you need to customize our packages you can do it with `kustomize`. It lets
-you create customized Kubernetes resources based on other Kubernetes resource
-files, leaving the original YAML untouched and usable as-is. To learn how to
-create your customization layer with it please see the `kustomize`
-[repository](https://github.com/kubernetes-sigs/kustomize).
+5. To deploy the packages to your cluster, execute:
 
-For further details please refer to the single package directories in this
-repository.
+```shell
+kustomize build . | kubectl apply -f -
+```
 
 ## Examples
 
 To see examples on how to customize Fury Kubernetes Monitoring packages, please
 go to [examples](examples) directory.
 
+<!-- Links -->
+
+[kube-prometheus-link]: https://github.com/prometheus-operator/kube-prometheus
+[prometheus-link]: https://github.com/prometheus/prometheus
+[alertmanager-link]: https://github.com/prometheus/alertmanager
+[servicemonitori-link]: https://github.com/prometheus-operator/prometheus-operator#customresourcedefinitions
```

Comment:
typo in `servicemonitori-link`, should be `servicemonitor-link` (there's an `i` that shouldn't be there)